index.md

```diff
@@ -1,62 +1,63 @@
 ---
 layout: event
 
-images:
-  - http://www.manilajs.com/images/015/share/share-e.png
-  - http://www.manilajs.com/images/015/share/share-f.png
-  - http://www.manilajs.com/images/015/share/share-d.png
-  - http://www.manilajs.com/images/015/share/see-you-later.png
+# images:
+#   - http://www.manilajs.com/images/015/share/share-e.png
+#   - http://www.manilajs.com/images/015/share/share-f.png
+#   - http://www.manilajs.com/images/015/share/share-d.png
+#   - http://www.manilajs.com/images/015/share/see-you-later.png
 
 event:
-  series: "015"
-  date: 2015-09-04
+  series: "016"
+  date: 2016-02-05
   time: 7PM
-  location: Silicon Valley Insight, BGC
+  location: VentureSpace PH, Shaw Boulevard
 
 registration:
-  date: 2015-08-03
-  state: over # announce | open | closed | over
-  signup: "http://goo.gl/forms/xH2LVnhmDl"
+  # date:
+  state: announce # announce | open | closed | over
+  # signup:
 
 hero:
   color: "#203a4f"
   opacity: 0.20
-  image: images/015/bg.jpg
-  # attribution_source: ""
-  # attribution_url: ""
+  image: images/016/bg.jpg
+  # :: current image is under Creative Commons Zero however,
+  #    so no attribution is required. Might as well though, yeah?
+  attribution_source: "NASA"
+  attribution_url: "https://unsplash.com/photos/NuE8Nu3otjo"
```

Comment:
TIL haha! Would you prefer we host the image ourselves, or just link from unsplash.it?